src/search/QuickOpenHelper.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2012 Adobe Systems Incorporated. All rights reserved.
```

Comment:
The copyright date for new files should be the year that they were created, so 2016 in this case.